### ISSUE TITLE
[Meru AB] [Blocked] Suggestions for Action screen (experimental)

### ISSUE THREAD

**Issue by philipperolet** [dust-tt/dust#3883]
Details on UX for these suggestions on  **[Figma](https://www.figma.com/file/QOxHwppG64GtPocpDhS6Y7/Product?type=design&node-id=4900-39294&mode=design&t=CrHO9TDk6FhlGRRq-4)** (also, see  context & discussion in the [making issue](https://github.com/dust-tt/decisions/issues/164))

- Help button next to the action
- from issue: `Assistant opening in a drawer specialized in helping the user make choices on Actions and Data Sources`
  - a new global assistant "builder_actions_help"
	- prompted to help with context above
	- with available data sources (names, top level folders, databases if any)
	- this should be done with templating in the global assistant prompt
  - should open with a first message
	- if there are instructions or a description, use it to suggest data sources, otherwise, ask what the assistant should be about (can be static)
	- should also suggest whether to use exhaustive or semantic, and the timeframe

- Ship with gating for dusters only, we'll iterate until good
- First iteration only works with data sources, not dust apps or databases
- Depends on #3783

**Comment by philipperolet** [comment]
@Duncid @gabhubert @spolu 
IMHO this task is quite chunky, and the benefit is lower than e.g. for instructions. More uncertain we'll get good suggestions too. I would recommend making it a nice to have

**Comment by Duncid** [comment]
Feels like it's uncertain thus we're backing off. It's going to be uncertain as long as we don't spend time on it.

**I think we should timebox 2h on it focused on just creating an assistant that would help a user figure out the right settings then decide if we want to do more.**

**Comment by gabhubert** [comment]
I'm +1 on investing here. We know users struggle (wrong settings or no assistants). But @Duncid is the proposal to invest 2h in framing?

**Comment by spolu** [comment]
blocked on first pass of experimentation on instructions

**Comment by philipperolet** [comment]
Closing, since nice-to-have, can be rescoped later as discussed IRL with @spolu
